### ISSUE TITLE
Add an audio file to video

### ISSUE THREAD

**Issue by pathumego** [mooniak/animate-svg-process#8]
Could be done using [Ffmpeg2theora](http://v2v.cc/~j/ffmpeg2theora/)

**Comment by lpsandaruwan** [comment]
I think ffmpeg also has the capability. I suggest less dependencies. But if you want this library to be included, let me know.

**Comment by pathumego** [comment]
No. Less dependencies better. If ffmpeg can do it that is more than enough. And if it is not much work lets implement it once we are done with the priority 1 set. 

Anyway this is not that important for us right now. We use Youtube and it has a service that you can add audio once file is uploaded.

**Comment by pathumego** [comment]
ffmpeg support is there for audio. So we try and implement a way to trim the audio and add it.
